### ISSUE TITLE
Arcticons normal & black are listed as themed icon source in Lawnchair

### ISSUE THREAD

**Issue by Kaiserdragon2** [Arcticons-Team/Arcticons#2402]
Just did Some Testing with our current release and these smartlauncher feature's were already working.
After some digging i found out that this line (already in the manifest) >
```
<!-- Lawnchair Launcher -->
<intent-filter>
      <action android:name="app.lawnchair.icons.THEMED_ICON" />
      <category android:name="android.intent.category.DEFAULT" />
</intent-filter>
```
makes the feature also work.
It also categorizes the iconpacks in the Lawnchair-Launcher as "Themed Icon Source" which the normal and black Arcticons are not.

_Originally posted by @Kaiserdragon2 in https://github.com/Arcticons-Team/Arcticons/issues/2095#issuecomment-2543141825_

**Comment by Donnnno** [comment]
Interesting, so we gotta split manifests for different versions somehow?

**Comment by Kaiserdragon2** [comment]
Yes that would be the case. This should be similar to our smartlauncher test.

**Comment by Kaiserdragon2** [comment]
Some helpful resources https://kvaesitso.mm20.de/docs/user-guide/customization/themed-icons.html
![image](https://github.com/user-attachments/assets/06e3ac0e-f02f-4e0f-b78e-4395a9a4924a)
The Debug ones are the Ones without the Lawnchair entry. These aren't themeable in Kvaesito.
On Lawnchair i couldn't make out a difference (only the listing).
Because this allows themable icons for Kvaesito on older Android versions i think we should leave the Android manifest as it is.
